pkg/ccl/backupccl/file_sst_sink.go

```diff
@@ -175,7 +175,8 @@ func (s *fileSSTSink) open(ctx context.Context) error {
 		}
 		s.out = e
 	}
-	s.sst = storage.MakeBackupSSTWriter(ctx, s.dest.Settings(), s.out)
```

Comment:
I think we might eventually rename it to `MakeTransportSSTWriter` and keep it around for at least a little while, since at least for now SST is the wire transport format for responses from kv to sql, where we really don't want to spend cycles on filters or block props.